docs/organizations/using-a-coding-standard.md

```diff
@@ -0,0 +1,95 @@
+
+    ![Creating a new coding standard](images/coding-standard-create.png)
+
+1.  Select all the programming languages that the new coding standard will cover and click **Next: Tools and patterns**.
+
+    The coding standard will only include configurations for the tools that support at least one of the selected languages.
+
+    ![Selecting the languages for the coding standard](images/coding-standard-select-languages.png)
+
+1.  Configure the tools and patterns of the coding standard and click **Next: Select and apply to repositories**.
+
+    -   Toggle the tools that Codacy will run when analyzing your code
+    -   For each enabled tool, configure the code patterns that the tool will use
+
+    You can use the filters to help you find the relevant tools and code patterns.
+
+    ![Configuring the tools and patterns for the coding standard](images/coding-standard-configure-tools.png)
+
+1.  Select existing repositories that should follow the new coding standard and click **Save and apply standard**.
+
+    Codacy will start using the new coding standard on the next analysis of each selected repository.
+
+    ![Applying the coding standard to repositories](images/coding-standard-apply.png)
+
+<!--NOTE
+    Add this section when CY-5181 is complete.
+
+## Setting a coding standard as default {: id="set-default"}
+
+New repositories in your organization will automatically follow the default coding standard.
+
+To set a coding standard as default:
+
+1.  Open your organization **Coding standards** page, tab **Coding standards**.
+
+1.  Toggle **Make default** on the coding standard card.
+
+    ![Setting a coding standard as the default](images/coding-standard-set-default.png)
+-->
+
+## Editing a coding standard {: id="editing"}
+
+To edit an existing coding standard or change the repositories that follow that coding standard:
+
+1.  Open your organization **Coding standards** page, tab **Coding standards**.
+
+1.  Click the edit icon on the coding standard card.
+
+    ![Editing an existing coding standard](images/coding-standard-edit.png)
+
+1.  Make your changes to the current coding standard configurations and click the button **Next** to advance between the following pages:
+
+    -   The programming languages that the coding standard applies to
+    -   The tool and code pattern configurations of the coding standard
+    -   The repositories that follow the coding standard
+
+1.  Click the button **Save and apply standard** on the repository selection page to save your changes to the coding standard.
+
+    Codacy will start using the updated coding standard on the next analysis of each selected repository.
+
+<!--NOTE
+    Add this tip card when CY-5181 is complete.
+
+!!! tip
+    To ensure that all new repositories automatically follow the coding standard, [set the coding standard as default](#set-default).
+-->
+
```

Comment:
What I'm missing or couldn't find in this page is anything about permissions. About only organization administrators being able to create, edit and apply standards. And also stating that repository owners can unlink a standard if they customize the code patterns at repository level.

Reply:
@alerizzo We have a dedicated page that covers everything related to permissions in synced organizations, and it's here that I'm mentioning that only administrators are able to manage coding standards:

https://deploy-preview-823--docs-codacy.netlify.app/organizations/roles-and-permissions-for-synced-organizations/

This page already mentioned who was allowed to configure code patterns at the repository level. If a repository is following a coding standard, can the same people who were already able to make changes to the code patterns also unlink the repository from the coding standard? Or only repository owners can unlink the repository from the coding standard? :thinking:

Reply:
I'd say yes, but @machadoit might know better!

Reply:
the same people who are able to make changes to code patterns will be able (through the actions of changing patterns/tools/parameters and using import patterns) to unlink the standard.

So in the current state, although only admins can create, manage and apply standards, unlinking a repository is in practice available for the permission level configured in "Organization Dashboard" -> "Member privileges" -> "Analysis Configuration"

Reply:
Thanks for the input @pedrocodacy, the current behavior was what I was expecting from a user standpoint. :+1:

I'm not going to change the permissions table for now since the current information is correct. However, if customers have questions about this detail we can make it more explicit.